content/en-us/designs/protocol-translation/_index.md

```diff
@@ -50,7 +50,7 @@ Aggregation or batching of data is not a part of this design as it is discussed
 The Gateway design contains [considerations]({{ <ref "/designs/gateway#what-approach-should-be-used-when-storing-messages-for-later-delivery" >}}) about how protocol translation logic can be extended with mechanisms that batch results or compute aggregations over them.
```

Comment:
Ugh, minor positioning needed @Bestehorn that I didn't see.

The format for "ref" needs to be `{{< ref "test"` instead of `{{ <ref "text"` at the beginning. The rest is correct.